parachain/pallets/basic-channel/src/outbound/test.rs

```diff
@@ -69,6 +69,7 @@ parameter_types! {
 }
 
 impl basic_outbound_channel::Config for Test {
+	type SourceId = AccountId;
```

Comment:
Is `AccountId` temporary  used for compatible with legacy codes and finally change to `ParaId` ? 
If so is it nessessary to also add `assetId` into `Message` structure to support MultiAsset from same parachain?

Reply:
Yes, using `AccountId` here purely to keep the tests passing. If we need to use another type for `SourceId`, we can update the runtimes.

I believe `assetId` will be part of the `Message.payload` field. The payload will be an encoded set of XCM messages that's opaque to the channels. Then if we want to send messages that don't need an `assetId`, we change the XCM messages and not the channel's `Message`.